content/registry/javascript-foxx-tracer.md

```diff
@@ -0,0 +1,14 @@
+---
+title: foxx-tracer - An instrumentation library for Foxx Microservices
+registryType: <instrumentation/tracer>
```

Comment:
* `<>` syntax is invalid
* is this a tracer or the instrumentation?  A tracer is something that _implements_ OpenTracing API, instrumentation is something that _uses_ the API to describe transactions.

Reply:
Sorry, this should be labelled a tracer. Sending an update shortly.